include/tvm/relay/transform.h

```diff
@@ -223,10 +223,12 @@ TVM_DLL Pass CombineParallelConv2D(uint64_t min_num_branches = 3);
  * `min_num_branch`.
  *
  * \param min_num_branches The minimun number of branches.
+ * \param to_batch_matmul Whether combine parallel dense ops to batch matmul.
```

Comment:
```suggestion
 * \param to_batch_matmul Whether to combine parallel dense ops to batch matmul.
```

Reply:
Hi, I don't quite understand here ` "CombineParallel instead of Batch"`, means we could just rename the pass like `BatchingOp` ?  I can work to wrap current 3 CombineParallelXXX pass into a sequential.

Reply:
@wrongtest exactly. I think there should just be a sequential wrapper so it is easier for ppl to call it.

Reply:
It is like 2 line of python code + 3 line of comment.